docs/development-mode.md

```diff
@@ -34,7 +34,7 @@ This is **not** how migrations are performed in development mode! When running i
 
 1. **Adjust data model**: Change your [data model definition](./data-modeling.md#data-model-definition) to match your desired database schema. Then **save** the schema file.
 
-Because the development server is watching your schema file (which includes the data model definition) in the background, it notices that you've performed a change and updates your database schema for you. There are **no migration files** created for this schema update! However, it does update Lift's `_Migrations` table in your database schema.
+Because the development server is watching your schema file (which includes the data model definition) in the background, it notices that you've performed a change and updates your database schema for you. There migration files created for this schema update are stored in the `migrations/dev` directory! However, it does update Lift's `_Migrations` table in your database schema.
```

Comment:
```suggestion
Because the development server is watching your schema file (which includes the data model definition) in the background, it notices that you've performed a change and updates your database schema for you. The migration files created for this schema update are stored in the `migrations/dev` directory. However, it does update Lift's `_Migrations` table in your database schema.
```